pandas/tests/indexing/test_loc.py

```diff
@@ -2067,7 +2067,10 @@ def test_loc_setitem_with_expansion_inf_upcast_empty(self):
         expected = Index([0, 1, np.inf], dtype=np.float64)
         tm.assert_index_equal(result, expected)
 
-    @pytest.mark.filterwarnings("ignore:indexing past lexsort depth")
+    @pytest.mark.filterwarnings(
+        "ignore:indexing past lexsort depth",
+        "ignore:Passing an empty string to Timestamp",
```

Comment:
Instead of ignoring in each of these tests, can you test the warning. See

https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#testing-a-warning